interfaces/apparmor/spec.go

```diff
@@ -524,7 +524,11 @@ func GenWritableMimicProfile(emit func(f string, args ...interface{}), path stri
 	emit("  # .. permissions for traversing the prefix that is assumed to exist\n")
 	for iter.Next() {
 		if iter.Depth() < assumedPrefixDepth {
-			emit("  \"%s\" r,\n", iter.CurrentPath())
+			cp := iter.CurrentPath()
+			if cp != "/" {
+				cp += "/"
+			}
```

Comment:
feels like maybe we should have CurrentPathPlusSlash as well to cover these cases here?